indico/modules/events/reviewing_questions_fields.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2017 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from wtforms.fields import BooleanField, StringField
+from wtforms.validators import DataRequired
+
+from indico.modules.events.fields import RatingReviewField
+from indico.util.i18n import _
+from indico.web.fields.base import BaseField, IndicoForm
+from indico.web.fields.simple import BoolField, TextField
+from indico.web.forms.widgets import SwitchWidget
+
+
+class BaseReviewingQuestionConfigForm(IndicoForm):
+    text = StringField(_('Question'), [DataRequired()])
+    is_required = BooleanField(_('Required'), widget=SwitchWidget())
```

Comment:
can we avoid replicating all the config forms here and just use the ones of the base fields? i think you are only replicating them to get rid of `description` but keeping a description wouldn't be so bad IMO! (on the review page we could show it as an info tooltip for example)

Reply:
This `title = StringField(_('Title'), [DataRequired()], description=_("The title of the field"))` from `FieldConfigForm` makes a little bit of a problem. Since in the context of reviewing fields  `Title` for the field label is wrong.

Reply:
"[Question] Title" is not so bad IMO, and you could modify the description after instantiating the form